src/main/java/world/bentobox/bentobox/Settings.java

```diff
@@ -109,6 +110,10 @@
     @ConfigComment("Time in seconds that players have to confirm sensitive commands, e.g. island reset")
     @ConfigEntry(path = "general.confirmation-time")
     private int confirmationTime = 20;
+    
+    @ConfigComment("Rank required to use a command. e.g., use the invite command. Default is owner rank is required.")
+    @ConfigEntry(path = "general.rank-command")
+    private Map<String, Integer> rankCommand = new HashMap<>();
```

Comment:
I don't understand how this would work : can you provide some default examples in the HashMap you're instantiating?

Reply:
Here's a copy from my config.yml:

```
  # Rank required to use a command. e.g., use the invite command. Default is owner rank is required.
  rank-command:
    /island team uncoop: 900
    /ai unban: 900
    /ai team kick: 500
    /ai ban: 500
    /ai team invite: 500
    /ai team untrust: 900
    /ai team coop: 900
    /island ban: 900
    /ai team trust: 900
    /island team coop: 900
    /island team untrust: 500
```

The key is unique because it's a fully-qualified command. The rank is the minimum rank value required, just like protection.